src/access/AccessControl.h

```diff
@@ -662,15 +662,20 @@ class AccessControl
      */
     CHIP_ERROR Check(const SubjectDescriptor & subjectDescriptor, const RequestPath & requestPath, Privilege requestPrivilege);
 
+    /**
+     * Validate whether an AccessControlEntryStruct is valid and complies with all defined constraints.
```

Comment:
needs rephrasing: parameter is `Entry` where as comment says `AccessControlEntryStruct`.

Does this need to be a member class or can it be static (and if it can be static, should it be a method on the `Entry` class instead)?